dvc/command/diff.py

```diff
@@ -200,9 +201,11 @@ def add_parser(subparsers, parent_parser):
         default=False,
     )
     diff_parser.add_argument(
+        "--md",
```

Comment:
I always try to use `-md` (single dash) on these abbreviations, so I am not a fan of this, but `--markdown` is too long. And `--md` is what we use in CML as well.